tyk-headless/values.yaml

```diff
@@ -139,6 +139,27 @@ gateway:
   #    hosts:
   #      - chart-example.local
 
+  ## Persistence parameters
+  ## ref: https://kubernetes.io/docs/user-guide/persistent-volumes/
+  ##
+  persistence:
+    ## Enable persistence on gateway using Persistent Volume Claims
+    ##
+    enabled: true
```

Comment:
i guess it'd be better to disable persistence by default. if users need to enable it, they can set this field to `true`